modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkCommand.java

```diff
@@ -188,7 +204,8 @@ public void setScroller(String scrollerName) {
          *
          * @param action the registered bulk action name
          * @param nxqlQuery by default an NXQL query that represents the document set to apply the action. When using a
-         *            generic scroller the query syntax is a convention with the scroller implementation.
+         *            generic scroller the query syntax is a convention with the scroller implementation. When using an
+         *            external scroller the field is the total number of elements if known.
```

Comment:
we need to rename nxqlQuery into query, but I don't see how the total number of elements for an external scroller is implemented

Reply:
It is not currently, and I will remove this part from the comment as the count is not known at the beginning of our scroller.